### PR TITLE
Update ko_kr.json

### PR DESCRIPTION
**Pull request by Merhaf**
:D

### CODE REVIEW

File: src/main/resources/assets/jade/lang/ko_kr.json [hunk 1]
```diff
@@ -10,75 +12,180 @@
...
+	"config.jade.display_mode_extra_msg": "toggle,hold,lite",
 	"config.jade.display_mode_toggle": "전환",
-	"config.jade.display_mode_toggle_desc": "${SHOW_OVERLAY} 키로 디스플레이를 켜고 끌 수 있습니다",
+	"config.jade.display_mode_toggle_desc": "[오버레이 표시] 키로 디스플레이를 켜고 끌 수 있습니다",
 	"config.jade.display_mode_hold_key": "유지",
-	"config.jade.display_mode_hold_key_desc": "${SHOW_OVERLAY} 키를 누르면 디스플레이가 표시됩니다",
+	"config.jade.display_mode_hold_key_desc": "[오버레이 표시] 키를 누르면 디스플레이가 표시됩니다",
 	"config.jade.display_mode_lite": "간단",
-	"config.jade.display_mode_lite_desc": "${SHOW_DETAILS} 키를 누르면 세부 정보가 표시됩니다",
+	"config.jade.display_mode_lite_desc": "[세부 정보 표시] 키를 누르면 세부 정보가 표시됩니다",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ko_kr.json]: All these `${SHOW_DETAILS}` and `${SHOW_OVERLAY}` should not be translated

File: src/main/resources/assets/jade/lang/ko_kr.json [hunk 2]
```diff
@@ -95,30 +202,114 @@
...
+	"jade.block_destroy_time": "파괴 시간: %s",
+	"jade.block_jump_factor": "점프 계수: %s",
+	"jade.block_speed_factor": "속도 계수: %s",
+	"jade.block_ignite_odds": "점화 확률: %s",
+	"jade.block_burn_odds": "연소 확률: %s",
+	"jade.ignore_list.comment": "이곳은 Jade의 타겟 무시 목록입니다. \"values\" 목록에 레지스트리 ID를 추가할 수 있습니다.",
+	"__comment": "테스트용:",
+	"jade.modName.cpp": "Crafting++",
+	"mymod.fuel": "연료: %d 틱",
+	"config.jade.plugin_debug.furnace_fuel": "테스트 연료"
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ko_kr.json]: Missing comma

File: src/main/resources/assets/jade/lang/ko_kr.json [hunk 3]
```diff
@@ -95,30 +202,114 @@
...
 	"jade.fluid.empty": "비어있음",
 	"config.jade.plugin_minecraft.lectern": "독서대",
+	"config.jade.plugin_minecraft.animal_owner": "동물 주인",
+	"config.jade.plugin_minecraft.animal_owner_extra_msg": "horse,pet",
+	"jade.owner": "주인: %s",
+	"jade.infinity": "+∞",
+	"jade.seconds": "{0}초",
+	"jade.minutes": "{0}m",
+	"jade.minutes_seconds": "{0}m {1}s",
+	"jade.error": "<%s 모드에서 오류 발생, 보고해주세요>",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ko_kr.json]: Please explicitly inform the user to report to the appropriate mod developer, i.e., translate the latest version as is

**Review by Snownee — CHANGES_REQUESTED** [review]

**Review by Merhaf — COMMENTED** [review]
Yes, I fixed it! :D

**Comment by Snownee** [comment]
Thank you!

**Pull request merged**